frontend/src/metabase/components/EntityMenuItem.styled.jsx

```diff
@@ -5,21 +5,21 @@ import { color } from "metabase/lib/colors";
 
 export const Item = styled.div`
   display: flex;
-align-items: center;
-  cursor: pointer;
-  color: ${color("text-medium")};
+  align-items: center;
+  cursor: ${props => !props.disabled && "pointer"};
```

Comment:
I'm a fan of the [not-allowed](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor#:~:text=not%2Dallowed.-,not,-%2Dallowed) pointer, especially here where the text color doesn't differ much for the disabled item.

```suggestion
  cursor: ${props => !props.disabled ? "pointer" : "not-allowed"};
```